tests/testJobQueue.py

```diff
@@ -93,11 +93,11 @@ def test_get(self):
         self.assertEqual(str(ret_job_2.id), self.jobId2)
 
     def test_getNextPendingJob(self):
-        self.jobQueue.assignJob(self.jobId2)
+        self.jobQueue.assignJob(self.jobId2, None)
```

Comment:
nit: change the header of `assignJob` to `def assignJob(self, jobId, vm = None)` so that you don't have to manually pass in None everytime you call it.